src/agentscope/models/dashscope_model.py

```diff
@@ -228,21 +231,28 @@ def __call__(
         if stream is None:
             stream = self.stream
 
+        # Update the kwargs with either 'app_id' or 'model' based on 'api_type'
         kwargs.update(
             {
-                "model": self.model_name,
                 "messages": messages,
                 # Set the result to be "message" format.
                 "result_format": "message",
                 "stream": stream,
+                "app_id"
+                if self.api_type == "application"
+                else "model": self.model_name,
```

Comment:
What's the value of `self.model_name` when `self.api_type == "application"`?

Reply:
Is `app_name` or `application_name` more appropriate when `api_type == "application"`? If so, consider to provide a new wrapper or add a new argument rather than directly using the argument `model_name`

Reply:
it does not need `model_name`, `app_id` determines which application to call.

Reply:
> Is `app_name` or `application_name` more appropriate when `api_type == "application"`? If so, consider to provide a new wrapper or add a new argument rather than directly using the argument `model_name`

ok, I'll change it to `app_id`.